### ISSUE TITLE
Feat. Request: add CW id string

### ISSUE THREAD

**Issue by gustavc39** [sm0svx/svxlink#587]
Hi,

I'd like to suggest an improvement in he CW id feature, to make it more flexible.  
The default behavior is to send (callsign)/R for repeaters and only (callsign) for link type logics.  The problem arises when some information needs to be added to that id string. For example, in my country (Argentina), the rules establish that the minimum data that must be included in the id of a repeater are the callsign and location (city).   Also, the addition of the "/R" string is not useful in our case and it must be removed to comply with regulations (in my country that would mean to be operating from a different province). 
While it's not difficult to make a custom Logic.tcl to accomplish this, it ends up being a hack, and if there's more than one repeater defined in the node, things get more complicated. 
In short, the suggestion is to add a new variable in the svxlink.conf fil to allow the user to define a string that is will be appended to the callsign to create the id.  The "/R" string could be the default value for that variable (in repeater type logics) to get the original behavior.
Thank you!

Gustavo - LU7WA

**Comment by dl1bz** [comment]
That's not really difficult. Make a new /usr/share/svxlink/events.d/local/Logic-mods.tcl and overwrite the procedures (e.g. **proc send_short_ident {{hour -1} {minute -1}}** ) you want to change. This is part of concept from svxlink for customizing things. If you do so, it's "update-safe".

I do so too and change only this part:
  # Play CW id if enabled
  if {$short_cw_id_enable} {
    puts "Playing short CW ID"
    if {$CFG_TYPE == "Repeater"} {
      set call "$mycall"
      playSilence 200;
      CW::play $call
      playSilence 200;
    } else {
      playSilence 200;
      CW::play $mycall
    }
    playSilence 500;
  }

73 Heiko, DL1BZ
